app/settings.py

```diff
@@ -75,6 +75,7 @@ def get_env_or_fail(key):
 EQ_DEV_MODE = parse_mode(os.getenv('EQ_DEV_MODE', 'False'))
 EQ_ENABLE_CACHE = parse_mode(os.getenv('EQ_ENABLE_CACHE', 'True'))
 EQ_ENABLE_FLASK_DEBUG_TOOLBAR = parse_mode(os.getenv('EQ_ENABLE_FLASK_DEBUG_TOOLBAR', 'False'))
+EQ_ENABLE_SECURE_SESSION_COOKIE = parse_mode(os.getenv('EQ_SECURE_SESSION_COOKIE', 'True'))
```

Comment:
The env var name here is incorrect so i'm not sure setting it with have any effect.

Reply:
Well spotted!